src/remotion/Typewriter/TypewriterTemplate.tsx

```
@@ -36,14 +68,55 @@ export const TypewriterTemplate: React.FC<TypewriterProps> = ({
+                    opacity: baseOpacity * (blink ? 1 : 0.3),
+                    content: '█',
+                    backgroundColor: 'transparent'
+                };
+            case 'underscore':
+                return { 
+                    opacity: baseOpacity * (blink ? 1 : 0.2),
+                    content: '_',
+                    backgroundColor: 'transparent'
+                };
+            case 'animated':
+                const pulse = Math.sin(frame / 10) * 0.5 + 0.5;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)